methods_test.go

```diff
@@ -305,6 +313,30 @@ var _ = Describe("Methods", func() {
 			Expect(response).ToNot(BeNil())
 			Expect(response.Status()).To(Equal(http.StatusOK))
 		})
+
+		It("Increments outbound prometheus counter", func() {
+			// Configure the server:
+			apiServer.AppendHandlers(
+				ghttp.RespondWith(http.StatusTeapot, nil, jsonHeader),
+			)
+
+			// Send the request:
+			response, err := connection.Post().
+				Path("/api/teapot/brew/coffee").
+				Send()
+			Expect(err).ToNot(HaveOccurred())
+			Expect(response).ToNot(BeNil())
+			Expect(response.Status()).To(Equal(http.StatusTeapot))
+
+			expectedLabels := prometheus.Labels{
+				"apiservice": "ocm-/api/teapot",
```

Comment:
@petli-openshift not sure if keeping the "/" was intended behavior of the fallback logic in `GetAPIServiceLabelFromPath()`, but that's how it works now.
If you want it to become e.g. `ocm--api-teapot`, I can send a separate PR.  

OK as-is IMHO, as the fallback is mostly there to draw attention to new services so we'll remember to add a new case to that function.

Reply:
yep, it's completely as is. thanks for the heads-up.